doc/how_to_start.rst

```diff
@@ -0,0 +1,72 @@
+..
+  SPDX-FileCopyrightText: 2021 The PyPSA meets Earth authors
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _how_to_start:
+
+
+##########################################
+How to run your model
```

Comment:
Why not tutorial?
This page is not just about running the model but also modifying and analysing it

Reply:
Fixed but avoided to focus on tutorial as the procedure explained remains valid in more general context, as well